CMakeLists.txt

```diff
@@ -555,19 +175,11 @@ set (CMAKE_C_FLAGS              "${CMAKE_C_FLAGS} -Wall -Wformat -Wformat-securi
 set (CMAKE_C_FLAGS_DEBUG        "${CMAKE_C_FLAGS_DEBUG} -Werror")
 set (CMAKE_C_FLAGS_RELEASE      "${CMAKE_C_FLAGS_RELEASE} ${HARDENING_FLAGS}")
 
+add_subdirectory (gsad)
```

Comment:
The 'gsad' subdirectory should be ignored when SKIP_SRC is defined. Otherwise all dependencies of gsad will be required to build documentation or packages.

Reply:
Ah good to know! I've just moved the check for SKIP_SRC to the new gsad/CMakeLists.txt. Wasn't aware of this problem. I'll update the PR.